app/api/admin_statistics_api/events.py

```diff
@@ -15,24 +15,38 @@
 @jwt_required()
 def event_topic_count():
     result_set = db.engine.execute(text(
-        "SELECT event_topics.name, COUNT(events.id) FROM event_topics " +
+        "SELECT event_topics.name AS name, event_topics.id AS id, " +
+        "COUNT(events.id) AS count FROM event_topics " +
         "LEFT JOIN events ON events.event_topic_id = event_topics.id " +
         "GROUP BY event_topics.id;"
     ))
-    event_topics_counts = dict(list(result_set))
-    return jsonify(event_topics_counts)
+    event_topics_count = []
+    for each in list(result_set):
+        event_topic = {}
+        event_topic["name"] = each[0]
+        event_topic["id"] = each[1]
+        event_topic["count"] = each[2]
+        event_topics_count.append(event_topic)
+    return jsonify(event_topics_count)
```

Comment:
Use the approach suggested previously

Reply:
Sure

Reply:
Done @iamareebjamal Please review.

![screenshot from 2018-06-09 18-25-23](https://user-images.githubusercontent.com/22936570/41193994-8567e43c-6c32-11e8-8cc9-679aa66b6fe2.png)
![screenshot from 2018-06-09 18-25-15](https://user-images.githubusercontent.com/22936570/41193995-85cfd5f6-6c32-11e8-8310-72a33014e6af.png)